hera_cal/abscal.py

```diff
@@ -389,11 +398,28 @@ def TT_phs_logcal(model, data, antpos, wgts=None, refant=None, verbose=True, zer
     echo("...finished linsolve", verbose=verbose)
 
     if not return_gains:
+        # rename variables ew/ns instead of 0/1 to maintain backwards compatability
+        if assume_2D:
+            params = list(fit.keys())
+            for p in params:
+                if 'Phi_0' in p:
```

Comment:
how do you know that Phi_0 is ew and not ns?

Reply:
Because of this sentence in the docstring: "If assume_2D is True, it is assumed that the [0] index contains the east-west separation and [1] index the north-south separation."